doc/releases/release-notes-2.1.rst

```diff
@@ -31,7 +31,27 @@ Boards & SoC Support
 Drivers and Sensors
 *******************
 
-* TBD
+* Added support for these ARM boards:
```

Comment:
I think this should be moved to 
"Boards & SoC Support" section

Reply:
OH! sorry :)

Reply:
Alright, fixed that @erwango